Sources/Public/CalendarView.swift

```diff
@@ -150,6 +172,10 @@ public final class CalendarView: UIView {
     }
   }
 
+  public override func layoutMarginsDidChange() {
+    setNeedsLayout()
```

Comment:
Feels like good practice to call super even if the docs say it doesn't do anything.